src/test/ui/native-library-link-flags/suggest-libname-only-1.stderr

```diff
@@ -0,0 +1,6 @@
+error: could not find native static library `libfoo.a`, perhaps an -L flag is missing?
+   |
+   = help: only provide the library name `foo`, not the full filename
```

Comment:
It's a shame we don't have span information here to point out the `link` attr or ideally the `"libfoo.a"`.